From efd0231aa7a68ec431ce99251d2496a0eed624ec Mon Sep 17 00:00:00 2001
From: Tim Parenti <tim@timtimeonline.com>
Date: Tue, 20 Oct 2020 10:53:09 -0400
Subject: [PATCH 6/6] Palestine ends DST on 2020-10-24

(Thanks to Sharef Mustafa.)
* asia (Palestine): Assume future fall transitions will be on the
Saturday preceding October's last Sunday at 01:00, with the recent
exception of the 2019-10-26 fall-back, which was at 00:00.
(Thanks to Steffen Thorsen.)
* NEWS: Mention this.
---
 NEWS | 18 ++++++++++++++++++
 asia | 28 ++++++++++++++++++++++++----
 2 files changed, 42 insertions(+), 4 deletions(-)

diff --git a/NEWS b/NEWS
index 25cf8cb..4d653dc 100644
--- a/NEWS
+++ b/NEWS
@@ -1,5 +1,23 @@
 News for the tz database
 
+Unreleased, experimental changes
+
+  Briefly:
+    Palestine ends DST earlier than predicted, on 2020-10-24.
+
+  Changes to future timestamps
+
+    Palestine ends DST on 2020-10-24 at 01:00, instead of 2020-10-31 as
+    previously predicted.  (Thanks to Sharef Mustafa.)  This affects
+    Asia/Gaza and Asia/Hebron.  Assume future fall transitions will be
+    on the Saturday preceding October's last Sunday.
+
+  Changes to past timestamps
+
+    Palestine's 2019-10-26 fall-back was at 00:00, not 01:00.  (Thanks
+    to Steffen Thorsen.)
+
+
 Release 2020c - 2020-10-16 11:15:53 -0700
 
   Briefly:
diff --git a/asia b/asia
index e4d84e9..1c2d3eb 100644
--- a/asia
+++ b/asia
@@ -3221,13 +3221,31 @@ Zone	Asia/Karachi	4:28:12 -	LMT	1907
 
 # From Sharef Mustafa (2019-10-18):
 # Palestine summer time will end on midnight Oct 26th 2019 ...
-# http://www.palestinecabinet.gov.ps/website/ar/ViewDetails?ID=43948
 #
+# From Steffen Thorsen (2020-10-20):
+# Some sources such as these say, and display on clocks, that DST ended at
+# midnight last year...
+# https://www.amad.ps/ar/post/320006
+#
+# From Tim Parenti (2020-10-20):
+# The report of the Palestinian Cabinet meeting of 2019-10-14 confirms
+# a decision on (translated): "The start of the winter time in Palestine, by
+# delaying the clock by sixty minutes, starting from midnight on Friday /
+# Saturday corresponding to 26/10/2019."
+# http://www.palestinecabinet.gov.ps/portal/meeting/details/43948
+
+# From Sharef Mustafa (2020-10-20):
+# As per the palestinian cabinet announcement yesterday , the day light saving
+# shall [end] on Oct 24th 2020 at 01:00AM by delaying the clock by 60 minutes.
+# http://www.palestinecabinet.gov.ps/portal/Meeting/Details/51584
+
 # From Paul Eggert (2019-04-10):
 # For now, guess spring-ahead transitions are March's last Friday at 00:00.
 #
-# From Tim Parenti (2016-10-19):
-# Predict fall transitions on October's last Saturday at 01:00 from now on.
+# From Tim Parenti (2020-10-20):
+# Predict future fall transitions at 01:00 on the Saturday preceding October's
+# last Sunday (i.e., Sat>=24).  This is consistent with our predictions since
+# 2016, although the time of the change differed slightly in 2019.
 
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
 Rule EgyptAsia	1957	only	-	May	10	0:00	1:00	S
@@ -3259,8 +3277,10 @@ Rule Palestine	2013	only	-	Sep	Fri>=21	0:00	0	-
 Rule Palestine	2014	2015	-	Oct	Fri>=21	0:00	0	-
 Rule Palestine	2015	only	-	Mar	lastFri	24:00	1:00	S
 Rule Palestine	2016	2018	-	Mar	Sat>=24	1:00	1:00	S
-Rule Palestine	2016	max	-	Oct	lastSat	1:00	0	-
+Rule Palestine	2016	2018	-	Oct	Sat>=24	1:00	0	-
 Rule Palestine	2019	max	-	Mar	lastFri	0:00	1:00	S
+Rule Palestine	2019	only	-	Oct	Sat>=24	0:00	0	-
+Rule Palestine	2020	max	-	Oct	Sat>=24	1:00	0	-
 
 # Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
 Zone	Asia/Gaza	2:17:52	-	LMT	1900 Oct
-- 
1.8.3.1

